examples/dependencies/child/flake.nix

```diff
@@ -0,0 +1,26 @@
+{
+  description = "Example project demonstrating project dependencies.";
+
+  inputs = {
+    pkgs.url = github:NixOS/nixpkgs/nixos-22.11;
+  };
+
+  outputs = { pkgs, ... }:
+    let
+      # TODO: not necessarily
```

Comment:
We should either use eachDefaultSystem or write that this example is not universal instead of having a TODO.

Reply:
Agreed